File: output2.js
// for (let angka = 1; angka <= 100; angka++) {
//     console.log("angka " + angka);
// }

for (let baris = 1; baris <= 5; baris++) {
    let hasil = "";

    for (let angka = 1; angka <= baris; angka++) {
        hasil += angka + " ";
    }

    console.log(hasil);
}

File: output2.cpp
// #include <iostream>
// using namespace std;

// int main() {
//     for (int angka = 1; angka <= 100; angka++) {
//         cout << "angka " << angka << endl;
//     }

//     return 0;
// }

#include <iostream>
using namespace std;

int main() {
    for (int baris = 1; baris <= 5; baris++) {
        for (int angka = 1; angka <= baris; angka++) {
            cout << angka << " ";
        }
        cout << endl;
    }

    return 0;
}